src/neuromancer/dynamics/integrators.py

```diff
@@ -162,21 +144,16 @@ def __init__(self, block, interp_u=None, h=1.0):
         """
 
         :param block: (nn.Module) A state transition model.
-        :param interp_u: Function for interpolating control input values for intermediate integration steps.
-                         If you assume a constant control sequence over the time intervals of the samples then
-                         lambda u, t: u will work.
-                         See interpolation.py and neuromancer/examples/system_identifcation/duffing_parameter.py for
-                         more sophisticated interpolation schemes.
         :param h: (float) integration step size
         """
         super().__init__(block=block, interp_u=interp_u, h=h)
 
-    def integrate(self, x, u, t):
+    def integrate(self, x, *args):
         h = self.h
-        k1 = self.block(self.state(x, t, t, u))                    # k1 = f(x_i, t_i)
-        k2 = self.block(self.state(x + h*k1/2.0, t + h/2, t, u))   # k2 = f(x_i + 0.5*h*k1, t_i + 0.5*h)
-        k3 = self.block(self.state(x + h*k2/2.0, t + h/2, t, u))   # k3 = f(x_i + 0.5*h*k2, t_i + 0.5*h)
-        k4 = self.block(self.state(x + h*k3, t + h, t, u))         # k4 = f(y_i + h*k3, t_i + h)
+        k1 = self.block(x, *args)                    # k1 = f(x_i, t_i)
+        k2 = self.block(x + h*k1/2.0, *args)         # k2 = f(x_i + 0.5*h*k1, t_i + 0.5*h)
```

Comment:
The old code modifies `t` to pass the `tq` arg to `self.state` for `k2` - `k4`. Does this need to be handled in the new implementation or is `tq` functionality being deprecated?

Reply:
If deprecating, update function def inline comments to remove `t_i` term

Reply:
Same comment applies to all other classes' `integrate` methods implemented below